auth-web/package.json

```diff
@@ -21,15 +21,15 @@
     "@sentry/integrations": "^5.27.2",
     "axios": "^0.21.1",
     "core-js": "^3.3.3",
-    "fas-ui": "git+https://github.com/bcgov/fas-ui.git#v1.0.2",
+    "fas-ui": "git+https://github.com/bcgov/fas-ui.git#v1.0.3",
     "http-status-codes": "^2.1.4",
     "keycloak-js": "^9.0.3",
     "mime-types": "^2.1.27",
     "moment": "^2.24.0",
     "regenerator-runtime": "^0.13.3",
     "register-service-worker": "^1.6.2",
     "sbc-common-components": "2.4.45",
-    "vue": "^2.6.12",
+    "vue": "2.6.12",
```

Comment:
I had to pin Vue to 2.6.12, because 2.6.13 has some weird type definitions that are different "util" property. 
The bcrs-shared-components uses 2.6.12.